.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
   "solidity.compileUsingRemoteVersion": "v0.7.5+commit.eb77ed08",
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "editor.formatOnSave": true
```

Comment:
This one might be a bit too optimistic - this can produce weird output when e.g. writing unit tests

Reply:
Why? If you have configured Prettier I don't see why this would be the case

Reply:
Let's try it out